src/cmd-init

```diff
@@ -15,13 +15,14 @@ Usage: coreos-assembler init --help
   For example, you can use https://github.com/coreos/fedora-coreos-config
   as GITCONFIG, or fork it.  Another option useful for local development
   (if you're running a shell inside this container) is to pass a file path
-  starting with `/` - a symlink to it will be created and then used directly."
+  starting with `/` - a symlink to it will be created and then used directly.
 EOF
 }
 
 # Call getopt to validate the provided input.
-options=$(getopt --options hf --longoptions help,force -- "$@")
-[ $? -eq 0 ] || {
+rc=0
+options=$(getopt --options hf --longoptions help,force -- "$@") || rc=$?
+[ $rc -eq 0 ] || {
```

Comment:
was there a good reason to switch to `rc` variable here?

Reply:
See the message of the first commit:

```
With `errexit` turned on, `getopt` would just cause the script to
immediately exit, so checking the exit code afterwards didn't actually
make any sense; it would always have been successful if it got to that
point.
```